### DISCUSSION TITLE
sponge and mud for clay

### DISCUSSION THREAD

**Discussion by RhythmicSys** [PurpurMC/Purpur#1049] [__Completed__] [answered]
![image](https://user-images.githubusercontent.com/45906780/178199247-f4859381-0ad7-4dc6-98eb-90d3824ecd9b.png)
with 1.19, to turn mud to clay you usually have to put it on dripstone and have it drip long enough to dry out
but I think it'd be neat to be able to place a sponge and have it soak up the water and turn it to clay like this:
![image](https://user-images.githubusercontent.com/45906780/178199577-ca1c14a0-7e74-4227-8475-111fc1b9b0e3.png)

**Comment by craftish37** [comment]
Can easily be done with a datapack or a plugin but why not 🤷‍♂️

**Reply by granny** [reply]
can't be done in a datapack without using functions (bad) and it's more annoying to implement through a plugin

**Comment by granny** [comment, marked as answer]
Implemented as of build #2032 under the `blocks.sponge.absorbs-water-from-mud` option. https://github.com/PurpurMC/Purpur/commit/76146eca495f8c70ceee841704656809b6c3fcaf
